larray/core.py

```diff
@@ -342,12 +346,16 @@ def _range_str_to_range(s):
 
     Examples
     --------
-    >>> list(_range_str_to_range('..3'))
-    [0, 1, 2, 3]
+    >>> list(_range_str_to_range('-1..2'))
+    [-1, 0, 1, 2]
     >>> _range_str_to_range('a..c')
     ['a', 'b', 'c']
     >>> list(_range_str_to_range('2..6 step 2'))
     [2, 4, 6]
+
+    any special character except . and spaces should work
+    >>> _range_str_to_range('a|+*@-b .. a|+*@-d')
+    ['a|+*@-b', 'a|+*@-c', 'a|+*@-d']
```

Comment:
so that you can write: "a+b .. a+d" or anything like that. The doctest example is obviously extreme, but the point was to test several of them in one go.